src/qtui/menus.cc

```diff
@@ -82,7 +87,11 @@ QMenuBar * qtui_build_menubar (QWidget * parent)
 
     static const audqt::MenuItem playback_items[] = {
         audqt::MenuCommand ({N_("_Play"), "media-playback-start", "Ctrl+Return"}, aud_drct_play),
+#ifdef Q_OS_MACOS
+        audqt::MenuCommand ({N_("Paus_e"), "media-playback-pause", "Space"}, aud_drct_pause),
```

Comment:
No. Space doesn't work in all parts of the GUI (not in any text entry, for example), which is why we have the alternate shortcut here.

Reply:
On Mac, Command+Comma opens the preferences dialog, so that wouldn't work here either. I'll see how easily I can get used to Command+Space .